lib/board.rb

```diff
@@ -67,14 +71,17 @@ def place(ship, coords)
 
   def render(show_ship = false)
     board = " "
-    letter_arr = ("A".."Z").to_a.slice(0,@max_board_size)
-
-    @max_board_size.times { |num| board += " #{num + 1}" }
+    letter_arr = ("A".."Z").to_a.slice(0, @max_board_size)
 
+    @max_board_size.times do |num|
+      board += "  #{num + 1}" if num < 10
+      board += " #{num + 1}" if num >= 10
+    end
+ 
     @cells.values.each_with_index do |cell, i|
 
       board << " \n" << letter_arr.shift if i % @max_board_size  == 0
-      board << " "
+      board << "  "
```

Comment:
### What does this do?

I gave everything in board a double space because introducing double digits made the board look wonky without it.